sql/executor.go

```diff
@@ -283,8 +284,12 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 		return parser.DBytes(t.BytesVal), true
 	case *driver.Datum_StringVal:
 		return parser.DString(t.StringVal), true
+	case *driver.Datum_DateVal:
+		return parser.DTimestamp{Time: t.DateVal.GoTime().UTC()}, true
```

Comment:
Actually, that's not the right thing to do. We shouldn't be blindly calling `UTC()` on everything; `UTC()` only changes the behaviour of `time.Time` for presentational purposes, not computational.

I've added a commit here to remove all the non-presentational uses of `UTC()`, PTAL.